package.json

```diff
@@ -129,18 +129,18 @@
     "numeral": "2.0.6",
     "pdf.js-extract": "0.0.5",
     "postcss": "5.2.16",
-    "prop-types": "0.2.0",
     "qrcode.react": "0.6.1",
     "random-words": "0.0.1",
     "react": "15.4.2",
     "react-addons-css-transition-group": "15.4.2",
     "react-copy-to-clipboard": "4.2.3",
-    "react-css-themr": "1.7.2",
+    "react-css-themr": "2.0.0",
```

Comment:
@DominikGuzei I had to fixate the version to `2.0.0` as newer ones throw errors and the app can't start...